CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to PaperMC Docs
+
+Thank you for considering contributing to the PaperMC Docs repository! We appreciate your help in making our documentation better. 
+To ensure a smooth and collaborative contribution process, please follow the guidelines below.
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [How to add a page](#adding-a-page)
+3. [Submitting Contributions](#submitting-contributions)
+4. [Style Guidelines](#style-guidelines)
+5. [Code of Conduct](#code-of-conduct)
+
+## Introduction
+
+The PaperMC Docs repository contains the documentation for the PaperMC project, which is used to provide essential 
+information, instructions, and guidelines to our users, developers, and contributors. Your contributions play a 
+significant role in improving and maintaining the quality of our documentation.
+
+We use Docosaurus to build and deploy the documentation website. The Docs Page is hosted on Vercel, which supports automatic
+CI/CD deployments from the main branch. The website is available at [https://docs.papermc.io](https://docs.papermc.io).
```

Comment:
Vercel also does deployments for all PR branches, though I'm not too sure how important it is to even mention vercel here, its not really material to contributions except that when you open a PR, vercel deploys that in a separate instance to you and others can view the site.